### PR TITLE
fix(ios): non large title navigation bars show default navigation bar

### PR DESCRIPTION
**Pull request by vijaysingh-axway**
https://jira.appcelerator.org/browse/TIMOB-27263

### CODE REVIEW

File: iphone/TitaniumKit/TitaniumKit/Sources/Modules/TiUIWindowProxy.m [hunk 1]
```diff
@@ -271,17 +271,24 @@ - (void)viewWillAppear:(BOOL)animated; // Called when the view is about to made
 #if __IPHONE_OS_VERSION_MAX_ALLOWED >= 130000
   if ([TiUtils isIOSVersionOrGreater:@"13.0"]) {
     TiColor *newColor = [TiUtils colorValue:[self valueForKey:@"barColor"]];
+    if (newColor == nil) {
+      //Get from TabGroup
+      newColor = [TiUtils colorValue:[[self tabGroup] valueForKey:@"barColor"]];
```
Comment (sgtcoolguy) [inline iphone/TitaniumKit/TitaniumKit/Sources/Modules/TiUIWindowProxy.m]: do you have to worry about [self tabGroup] being nil?

Comment (vijaysingh-axway) [reply iphone/TitaniumKit/TitaniumKit/Sources/Modules/TiUIWindowProxy.m]: No. In Objective C it is fine to pass message on nil object. It'll return nil.

**Comment by lokeshchdhry** [comment]
@vijaysingh-axway , On IOS 13, the properties like:
```
font: {fontFamily:'Snell Roundhand', fontSize:50},
shadow:{color:'gray', offset:{width:40,height:1}}},
```
are not respected. On IOS 12.2 they are.

Screen Shot 2019-09-09 at 9 27 12 AM

**Comment by lokeshchdhry** [comment]
FR Passed.

`titleAttributes` are now respected.

Studio Ver:       5.1.4.201909061933
SDK Ver:          8.2.0 local build
OS Ver:           10.14.5
Xcode Ver:        Xcode 11.0
Appc NPM:         4.2.15
Appc CLI:         7.1.1
Daemon Ver:       1.1.3
Ti CLI Ver:       5.2.1
Alloy Ver:        1.14.1
Node Ver:         10.16.1
NPM Ver:          6.9.0
Java Ver:         10.0.2
IOS Simulator:    ⇨ iPhone Xʀ iOS 13.0

**Pull request merged**